internal/bgp/bgp.go

```diff
@@ -665,3 +679,30 @@ func buildTCPMD5Sig(addr net.IP, key string) tcpmd5sig {
 
 	return t
 }
+
+// localAddressExists returns true if the provided address exists locally on
+// any of the host's network interfaces. The function doesn't return errors and
+// instead returns false if an error has occurred while looking for the
+// address. This function is safe to call if addr is nil or malformed.
+func localAddressExists(addr string) bool {
```

Comment:
This function is missing logs. It will be hard to diagnose any problems otherwise

Reply:
True, and thanks for noticing. For some reason I thought the entire package doesn't have logging and didn't want to expand the scope of this PR by adding logging. Now I see that I can easily add logging to this function.